alfred/files/alfred.init

```diff
@@ -103,41 +94,38 @@ alfred_start()
 	return 0
 }
 
-start()
-{
+start_service() {
 	config_load "alfred"
 	config_foreach alfred_start alfred
-
-	if [ "$enable" = "0" ]; then
-		exit 0
-	fi
-
-	echo "${initscript}: starting alfred"
-	SERVICE_PID_FILE="$pid_file_alfred"
-	service_start /usr/sbin/alfred ${alfred_args}
-
-	if [ "$vis_enable" = "1" ]; then
-		echo "${initscript}: starting batadv-vis"
-		SERVICE_PID_FILE="$pid_file_vis"
-		service_start /usr/sbin/batadv-vis ${vis_args}
-	fi
-
-	if [ "$run_facters" = "1" ]; then
+	procd_open_instance "alfred"
+	echo "starting alfred"
```

Comment:
Why this echo? Other procd scripts also don't do this.

Reply:
@ecsv so if you like I'd remove that?

Reply:
I just find it odd that it is there. I am neither the maintainer nor have I write access to the repository.

Reply:
@aparcar please remove it. Thanks.

Reply:
this here as well? https://github.com/openwrt-routing/packages/pull/390/files/a75b388cea308b8867b2d45a6ab39be89c4847f8#diff-9cac72d591240485e6d7dd48f4aaa90eR108